### PR TITLE
Adds Riot Launcher, new 40mm grenade variants, and new stalker magazine sprites.

### PR DESCRIPTION
**Pull request by PerfectTangent**
### Purpose

This PR adds the Riot Launcher, a single shot 40mm grenade launcher at the disposal of security to be stored in the lethals locker. It also adds 40mm riot and canister shells ready to use and includes the sprites for smoke shells down the road. This PR also provides new distinct color-striped variants of the stalker's specialist ammo mags.

Everything tested in editor.

### Notes:

This PR:
- adds the 40mm Riot Launcher
- adds 40mm Riot shells. They are essentially huge rubber bullets that stun for significantly longer and deal more lethal damage.
- adds 40mm Canister shells; a 10-pellet buckshot shell with huge spread.
- adds new magazine sprites for c20r.45 AP, HP, and Incendiary mags.
- updates the sprites for the basic 40mm HE shell and box, and adds new shells and boxes for the new 40mm ammo variants.

The shitload of added sprites is a result of the many different c20r magazine states that exist. God, there's so many. Spriters need to practice using sprite overlays more.

Sprotes done by me, though I was mostly working off of the existing ammo sprites.

### Changelog:

CL: [New] 40mm Riot Launcher added for sec. Can currently fire HE, Canister, and Riot shells.
CL: [Improvement] c20r.45 stalker magazines now have color-striped versions for the AP, HP, and Incendiary ammo types.

### CODE REVIEW

**Review by NoooneyDude — DISMISSED** [review]

File: UnityProject/Assets/Resources/ScriptableObjectsSingletons/SpriteCatalogueSingleton.asset [hunk 1]
```diff
@@ -23561,3 +23561,12 @@ MonoBehaviour:
...
   - {fileID: 11400000, guid: 8c4c7a5028a4e99428a8c7b87e7ba608, type: 2}
+  - {fileID: 11400000, guid: ad7fc6445ea7c4544b5a72bd4c4afeb5, type: 2}
+  - {fileID: 11400000, guid: 4219400b541a6a349b25f9ce17296db7, type: 2}
+  - {fileID: 11400000, guid: 0b4333678974d01408da4debbfdedc9e, type: 2}
+  - {fileID: 11400000, guid: 32b454f4a7959174c84a6bd74c239865, type: 2}
+  - {fileID: 11400000, guid: 9cad2390beccbaf4da62d9db5f30d723, type: 2}
+  - {fileID: 11400000, guid: a1f98561cb382d346b991dee26d6134f, type: 2}
+  - {fileID: 11400000, guid: bae93943baf19e845b7bf2f344e8090d, type: 2}
+  - {fileID: 11400000, guid: 0f15becefc908324b9a1cad24593c8bf, type: 2}
+  - {fileID: 0}
```
Comment (NoooneyDude) [inline UnityProject/Assets/Resources/ScriptableObjectsSingletons/SpriteCatalogueSingleton.asset:23572]:
```suggestion
```

**Comment by PerfectTangent** [comment]
great, now it's conflicting.

**Comment by corp-0** [comment]
I think the way I'd solve this cojnflict is by resetting SpriteCatalogueSingleton.asset to latest develop and readd the new sprites to the catalogue which I think is automatic(?)

``git checkout upstream/develop -- UnityProject/Assets/Resources/ScriptableObjectsSingletons/SpriteCatalogueSingleton.asset``

**Comment by PerfectTangent** [comment]
ain't automatic, and when I open the catalogue to add them, the newly added entries become null because they are pointing to SpriteData SOs that don't exist in my branch.

**Comment by PerfectTangent** [comment]
given that we've found there are other missing SOs from the list, I think we should merge this then regenerate the catalogue.

**Comment by corp-0** [comment]
We can't merge something with conflicts. You have to reset the file and rebase from upstream/develop to get the new commits into your branch

**Comment by PerfectTangent** [comment]
ayyy, it passed.

**Review by NoooneyDude — APPROVED** [review]

**Review by corp-0 — APPROVED** [review]

**Pull request merged**